src/renderer/entities/transaction/ui/QrCode/QrReader/QrReaderWrapper.tsx

```diff
@@ -118,91 +118,92 @@ export const QrReaderWrapper = ({ className, onResult, countdown, validationErro
   };
 
   return (
-    <div
-      className={cnTw(
-        'relative flex w-full flex-1 flex-col items-center overflow-y-hidden pt-[52px]',
-        isLoading && 'bg-black',
-      )}
-    >
-      <SmallTitleText as="h3" className={cnTw('z-10', isCameraOn && 'text-white')}>
-        {t('signing.scanQrTitle')}
-      </SmallTitleText>
-
-      <Countdown countdown={countdown} />
-
-      {/* scanning frame */}
-      <div className="mb-4 h-[240px] w-[240px]">
-        <div className="relative">
-          <div
-            className={cnTw(
-              'absolute z-20 h-[240px] w-[240px]',
-              isCameraOn ? (isSuccess ? 'border-text-positive' : 'border-white') : 'border-filter-border',
-              'rounded-[22px] border-2',
-            )}
-          ></div>
-          <div className="absolute z-30 flex h-[240px] w-full flex-col items-center justify-center gap-y-4">
-            <SignatureReaderError error={error} isCameraOn={isCameraOn && !isLoading} onTryAgain={onRetryCamera} />
+    <ThemeProvider theme="dark">
+      <div
+        className={cnTw(
+          'relative flex w-full flex-1 flex-col items-center overflow-y-hidden pt-[52px]',
+          isLoading && 'bg-black',
+        )}
+      >
+        <SmallTitleText as="h3" className={cnTw('z-10', isCameraOn && 'text-white')}>
+          {t('signing.scanQrTitle')}
+        </SmallTitleText>
+
+        <Countdown countdown={countdown} />
+
+        {/* scanning frame */}
+        <div className="mb-4 h-[240px] w-[240px]">
+          <div className="relative">
+            <div
+              className={cnTw(
+                'absolute z-20 h-[240px] w-[240px]',
+                isCameraOn ? (isSuccess ? 'border-text-positive' : 'border-white') : 'border-filter-border',
+                'rounded-[22px] border-2',
+              )}
+            ></div>
+            <div className="absolute z-30 flex h-[240px] w-full flex-col items-center justify-center gap-y-4">
+              <SignatureReaderError error={error} isCameraOn={isCameraOn && !isLoading} onTryAgain={onRetryCamera} />
+            </div>
           </div>
-        </div>
 
-        {isLoading && <Shimmering width={240} height={240} className="absolute rounded-[22px]" />}
+          {isLoading && <Shimmering width={240} height={240} className="absolute rounded-[22px]" />}
```

Comment:
maybe replace it with `Skeleton`?

Reply:
It has some weird position and styling, let's keep it for now